etc/DependencyInstaller.sh

```diff
@@ -9,7 +9,7 @@ _versionCompare() {
 }
 
 _equivalenceDeps() {
-    yosysVersion=yosys-0.33
+    yosysVersion=yosys-0.42
```

Comment:
@maliberty Shouldn't there be a single source of truth here?

Yosys is both a submodule and there is a version number in the install files. Which is the single source of truth?

```
$ git submodule foreach pwd
Entering 'LSOracle'
/home/oyvind/OpenROAD-flow-scripts/tools/LSOracle
Entering 'OpenROAD'
/home/oyvind/OpenROAD-flow-scripts/tools/OpenROAD
Entering 'yosys'
/home/oyvind/OpenROAD-flow-scripts/tools/yosys
$ git describe 
yosys-0.36-782-g9b6afcf3f
```

Reply:
eqy is not a submodule of OR (nor yosys)

Reply:
Quite. I recall.

I get errors locally after running normal setup on eqy. Unlike yosys and openroad the correct version is not added to the path when I run ". env.sh"